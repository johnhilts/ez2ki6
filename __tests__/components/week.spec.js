describe('Week component snapshot test', () => {
  it('Should compare the Week component with a snapshot', () => {
    const component = '<div>Hello Jest</div>'
    expect(component).toMatchSnapshot();
  })
});
