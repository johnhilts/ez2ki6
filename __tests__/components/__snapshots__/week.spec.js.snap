exports[`Week component snapshot test Should compare the Week component with a snapshot 1`] = `
<div>
  <div
    className="row"
    style={
      Object {
        "verticalAlign": "middle",
      }
    }>
    <div
      className="col-sm-4 col-lg-3"
      style={
        Object {
          "verticalAlign": "middle",
        }
      }>
      <h2>
        <a>
          <<
        </a>
         
        Week # undefined
         
        <a>
          >>
        </a>
      </h2>
    </div>
  </div>
  <div
    className="row"
    style={
      Object {
        "verticalAlign": "middle",
      }
    }>
    <table
      cellPadding="10"
      cellSpacing="10">
      <thead>
        <tr>
          <th
            style={
              Object {
                "textAlign": "center",
              }
            }>
            Sunday
          </th>
          <th
            style={
              Object {
                "textAlign": "center",
              }
            }>
            Monday
          </th>
          <th
            style={
              Object {
                "textAlign": "center",
              }
            }>
            Tuesday
          </th>
          <th
            style={
              Object {
                "textAlign": "center",
              }
            }>
            Wednesday
          </th>
          <th
            style={
              Object {
                "textAlign": "center",
              }
            }>
            Thursday
          </th>
          <th
            style={
              Object {
                "textAlign": "center",
              }
            }>
            Friday
          </th>
          <th
            style={
              Object {
                "textAlign": "center",
              }
            }>
            Saturday
          </th>
        </tr>
      </thead>
      <tbody>
        <tr>
          <td
            style={
              Object {
                "borderStyle": "solid",
                "borderWidth": 5,
                "paddingBottom": 25,
                "paddingLeft": 25,
                "paddingRight": 25,
                "paddingTop": 25,
                "textAlign": "center",
                "verticalAlign": "top",
                "width": 100,
              }
            }>
            <a
              onClick={[Function]}
              style={Object {}}>
              <div
                style={
                  Object {
                    "fontWeight": "normal",
                  }
                }>
                15
                
                
              </div>
            </a>
          </td>
        </tr>
      </tbody>
    </table>
  </div>
</div>
`;
